package.json

```diff
@@ -6,20 +6,21 @@
     "develop": "npm-run-all -s ensure-env start-develop",
     "ensure-env": "node ./tools/scripts/ensure-env.js",
     "lint": "echo 'Warning: TODO - Define Linting.'",
-    "pretest": "npm-run-all -s lint bootstrap",
+    "pretest-ci": "npm-run-all -s lint bootstrap",
+    "seed": "node tools/scripts/seed/seedChallenges",
```

Comment:
This will require a change on the docs branch to remove the `node seed` option in the quick reference.